src/Sylius/Behat/Page/SymfonyPage.php

```diff
@@ -77,6 +77,25 @@ protected function waitForModalToAppear(NodeElement $modalContainer)
     }
 
     /**
+     * @param NodeElement $collection
+     *
+     * @return NodeElement|bool
```

Comment:
Can you refactor this method as @Zales0123 mention in [comment](https://github.com/Sylius/Sylius/pull/4499/files#r57282899)?